src/drag_data.pyx

```diff
@@ -45,10 +67,6 @@ cdef class DragData:
                 raise Exception("Image is not available")
             return PyImage_Init(cef_image)
 
-        cpdef tuple GetImageHotspot(self):
```

Comment:
You can't remove that function. This must be restored. I assume this is related to you altering CEF version in cef_version_win.h.